ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1260,12 +1260,13 @@ periodics:
     workdir: true
   labels:
     ci.openshift.io/role: infra
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-prow-auto-testgrid-generator
   spec:
     containers:
     - args:
       - --github-token-path=/etc/github/oauth
-      - --github-endpoint=https://api.github.com
+      - --github-endpoint=http://ghproxy
```

Comment:
Looks like our other jobs keep the direct URL as a backstop when the proxy is down, so we should do it here, too:

```
       - --github-endpoint=http://ghproxy                                                                                                                                                 
       - --github-endpoint=https://api.github.com
```

Reply:
Will just adding this work, ie won't I need to make changes in the code to support this backstop behavior?

Reply:
No, that is the behaviour already with multiple entries